baseplate.go

```diff
@@ -128,6 +131,11 @@ type ServeArgs struct {
 func Serve(ctx context.Context, args ServeArgs) error {
 	server := args.Server
 
+	setRuntimeState := func(runtimebp.State) { /*noop*/ }
+	if v, ok := server.Baseplate().(interface{ setState(runtimebp.State) }); ok {
```

Comment:
🔕 
I'm not 100% sure about this, but I feel like `setState` should just be a hidden part of the `Baseplate` interface, especially since `State` won't work properly without it. Looking back, we probably should have just made `Baseplate` a struct with non-public fields rather than an interface with a hidden implementation where we would have likely just done it this way to begin with 🤷